frontend/src/components/About.jsx

```
@@ -0,0 +1,141 @@
+        {/* Content with two columns for larger screens */}
+        <div className="grid grid-cols-1 md:grid-cols-12 gap-8 items-center">
+          {/* Left column with illustration */}
+          <motion.div
+            className="md:col-span-5 flex justify-center"
+            initial={{ opacity: 0, x: -50 }}
+            whileInView={{ opacity: 1, x: 0 }}
+            transition={{ duration: 0.8 }}
+            viewport={{ once: true }}
+          >
+            <div className="relative">
+              <div className="absolute -inset-4 rounded-full bg-gradient-to-r from-purple-200 to-indigo-100 opacity-50 blur-lg"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)